komga-webui/src/locales/en.json

```diff
@@ -274,7 +274,7 @@
     },
     "delete_collection": {
       "button_confirm": "Delete",
-      "confirm_delete": "Yes, delete the collection \"{name}\"",
```

Comment:
Preference. Usually italics when referenced after.

Reply:
> Usually italics when referenced after.

do you mean italics like _italics_ ? This has nothing to do with translations though?

Reply:
Yes it does, because of localization.
We have avoided said problem.

Reply:
Using formatting within the translated strings brings its lot of complexity at code level, namely the need to use raw HTML strings instead of normal strings, I am trying to limit their use as much as possible.